container/container_unix.go

```diff
@@ -343,6 +343,17 @@ func (container *Container) BuildCreateEndpointOptions(n libnetwork.Network) ([]
 		createOptions = append(createOptions, libnetwork.EndpointOptionGeneric(genericOption))
 	}
 
+	/* FIXME: When restoring, the container should get the same IP address it had before, so
```

Comment:
At the moment, I believe that approach isn't compatible with CRIU. @xemul can correct me if I'm wrong, but I believe CRIU is going to force the container to have the same IP that existed during checkpoint and there's no current way to disable that.